src/shared/components/pipeline-run-logs/PipelineRunLogs.scss

```diff
@@ -11,6 +11,9 @@
     position: relative;
     top: 0.35em;
   }
+  &__navitem {
+    cursor: pointer;
```

Comment:
Do we know why the cursor isn't pointer already? PF NavItem should render an `<a>` and thus have the pointer cursor already.

Reply:
gotcha